README.md: update link to YP docs

gizero opened the pull request:
Update Yocto Project documentation link to point to latest released Quick Start Guide (2.2.1).

gizero committed:
Update README.md

Update Yocto Project documentation link to point to latest released Quick Start Guide (2.2.1).

rewitt1 commented on README.md:
````diff
@@ -63,4 +63,4 @@ point.
   ```
   At this point you should be able to follow the same instructions as described
-  in https://www.yoctoproject.org/docs/2.1/yocto-project-qs/yocto-project-qs.html#releases.
+  in https://www.yoctoproject.org/docs/2.2.1/yocto-project-qs/yocto-project-qs.html#releases.
````
I think we should just change the 2.2.1 to "latest" and then we'll never have to worry about this again.

Please add a signed off line as well and I'll merge it.

bavery22 replied:
latest works for me.

gizero replied:
Thought you wanted to reference the most recent **released** documentation here: _latest_ links point to ongoing builds of documentation. They may reference, as is the case these days with 2.3, a yet unreleased version of the Yocto project. If we don't mind this could appear misleading to newcomers (people who probably could benefit most of this link indeed), I agree using _latest_ is definitely more self-maintaining.

rewitt1 commented:
That's a good point. If we want to stick with released versions, then I
think "current" would work instead of "latest".

On Apr 12, 2017 10:24 PM, "Andrea Galbusera" <notifications@github.com>
wrote:

> *@gizero* commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/crops/poky-container/pull/13#discussion_r111313622>:
>
> > @@ -63,4 +63,4 @@ point.
>    pokyuser@3bbac563cacd:/workdir$
>    ```
>    At this point you should be able to follow the same instructions as described
> -  in https://www.yoctoproject.org/docs/2.1/yocto-project-qs/yocto-project-qs.html#releases.
> +  in https://www.yoctoproject.org/docs/2.2.1/yocto-project-qs/yocto-project-qs.html#releases.
>
> Thought you wanted to reference the most recent *released* documentation
> here: *latest* links point to ongoing builds of documentation. They may
> reference, as is the case these days with 2.3, a yet unreleased version of
> the Yocto project. If we don't mind this could appear misleading to
> newcomers (people who probably could benefit most of this link indeed), I
> agree using *latest* is definitely more self-maintaining.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/crops/poky-container/pull/13#discussion_r111313622>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADfcu0mwvR4ZH7MpsO7Uvm_QbxQRSKK0ks5rvbGLgaJpZM4M2pGd>
> .
>

gizero commented:
Right! Didn't know about _current_. That's exactly what we need... I'll send an updated PR

gizero committed:
Update README.md

Update Yocto Project documentation link to point to current released Quick Start Guide, by avoiding hardcoded version numbers.

rewitt1 reviewed (changes requested):
Could you add a "Signed-off by" to the commit message please?

gizero commented:
Closing this and re-sending as a new PR ( #15 ) with a single patch and "Signed-off by"
